### ISSUE TITLE
Mova para  <th:block layout:fragment="js-view">

### ISSUE THREAD

**Issue by rodrigonoll** [lab-ds/crm-lab-team#35]
https://github.com/lab-ds/crm-lab-team/blob/d1fab8724f4ca94237782675d9a2887e2e92cb89/src/main/resources/templates/account/list.html#L118-L122

**Comment by rodrigonoll** [comment]
Veja https://github.com/lab-ds/crm-lab-team/blob/2e3b44998f52fd5baf22fec57991ca98c80b5f55/src/main/resources/templates/task/list.html#L115-L116

**Comment by NikolasDaroit** [comment]
```
<th:block layout:fragment="js-view"> 

	<th:block th:include="account/fragments/cep :: zipFill"></th:block>

</th:block>
```
@rodrigonoll rearranjo desta maneira ou coloco o código do zipFill dentro do block js-view?

**Comment by rodrigonoll** [comment]
Olá,

Extraia o código js para um arquivo js e o referencie no th:block js-view.

Em ter, 14 de mai de 2019 22:22, NikolasDaroit <notifications@github.com>
escreveu:

**Comment by NikolasDaroit** [comment]
Closed on https://github.com/lab-ds/crm-lab-team/commit/3d49793a3f7af4adb8faaaf6adfc9b252d0b96eb

**Comment by NikolasDaroit** [comment]
Resolved on 13c874f988c20de7c33c7b6f2f01fbd64c66bb2a
